ui/app/AppLayouts/Profile/views/DevicesView.qml

```diff
@@ -21,17 +21,18 @@ SettingsContentBase {
 
     property bool isSyncing: false
 
-    Item {
+    Component.onCompleted: {
+        root.devicesStore.onSectionSetActive()
```

Comment:
This part will lead to confusion. When you make `DevicesView` visible/active multiple times, `onSectionSetActive` is triggered only once. Please consider renaming it to what it really does.

```suggestion
        root.devicesStore.loadDevices()
```

Reply:
Great point. Updated